action.yaml

```diff
@@ -6,52 +6,75 @@ description:
   found to be resolved, allows the previously added comment, if one exists at
   that time, to be removed from the PR.
 inputs:
-  instruction-text:
+  pull-request-number:
+    required: true
+    description: Uniquely identifies the Pull Request of interest.
```

Comment:
Are we not able to pull this from the github runner context?

Reply:
Good point, it appears we can yes, so I will remove this input from the action.

Reply:
That input has been removed, I will mark this point as being resolved.